ports/atmel-samd/README.rst

```diff
@@ -113,6 +113,18 @@ PB02                   **Yes**                         **Yes**  **Yes**  **Yes**
 PB03                   **Yes**                         **Yes**  **Yes**  **Yes**             **Yes**    **Yes**  **Yes**                 **Yes**    **Yes**               **Yes**       **Yes**           **Yes**
 =====================  ========  =========  =========  =======  =======  =======  =========  =========  =======  ==========  ==========  =========  =========  =========  ============  =======  =======  =========
 
+Setup
+-----
+Install required compiler packages:
+
+.. code-block:: shell
+
+    sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa
+    sudo dpkg --add-architecture i386
+    sudo apt-get update -qq || true
+    sudo apt-get install -y python3 gcc-multilib pkg-config libffi-dev libffi-dev:i386 qemu-system gcc-mingw-w64
+    sudo apt-get install -y --force-yes gcc-arm-embedded
```

Comment:
Same as #461:

I don't think the middle three are actually required. Please also remove the -y and --force-yes arguments since these will be manually run.